helm/cloud-provider-azure/values.yaml

```diff
@@ -77,3 +77,8 @@ cloudNodeManager:
     requestsMemWin: "50Mi"
     limitsCPUWin: "2"
     limitsMemWin: "512Mi"
+  enableHealthProbeProxy: false
+  healthCheckPort: 10356
+  targetPort: 10256
+  healthProbeProxyImage: "ss104301/health-probe-proxy:latest"
```

Comment:
Use my personal image as a placeholder and will replace to the mcr image when they are ready.